src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -348,6 +348,36 @@ private void testPublishUnpublishActions() {
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_PUBLISHED);
         assertTrue(BackDoor.getFeedbackSession(feedbackSessionPublished.getCourseId(),
                                                feedbackSessionPublished.getFeedbackSessionName()).isPublished());
+
+        ______TS("resend links action: PUBLISHED feedback session");
+        homePage.verifyResendPublishedLinksButtonExist(feedbackSessionPublished.getCourseId(),
+                                                       feedbackSessionPublished.getFeedbackSessionName());
+        homePage.clickSessionResultsOptionsCaretElement(feedbackSessionPublished.getCourseId(),
+                                                        feedbackSessionPublished.getFeedbackSessionName());
+        homePage.clickResendPublshedLinksLink(feedbackSessionPublished.getCourseId(),
+                                              feedbackSessionPublished.getFeedbackSessionName());
+        homePage.cancelResendPublshedLinksForm();
+        homePage.clickSessionResultsOptionsCaretElement(feedbackSessionPublished.getCourseId(),
+                feedbackSessionPublished.getFeedbackSessionName());
+        homePage.clickResendPublshedLinksLink(feedbackSessionPublished.getCourseId(),
+                                              feedbackSessionPublished.getFeedbackSessionName());
+        homePage.waitForAjaxLoaderGifToDisappear();
+        homePage.submitResendPublshedLinksForm();
+        homePage.waitForPageToLoad();
+        homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_RESEND_LINKS_EMPTY_RECIPIENT);
+        homePage.clickSessionResultsOptionsCaretElement(feedbackSessionPublished.getCourseId(),
```

Comment:
Similar here, if this is a new test case, perhaps separate them and add some comments

Reply:
done 👍